patroni/dcs/kubernetes.py

```diff
@@ -746,7 +746,8 @@ def is_ready(self) -> bool:
 
 class Kubernetes(AbstractDCS):
 
-    _CITUS_LABEL = 'citus-group'
+    # For backward compatibility, we keep citus-group here.
+    _MPP_GROUP_LABEL = 'citus-group'
```

Comment:
Should we maybe introduce another property in AbstractMPP for that?
```python
@property
def k8s_group_label(self):
    return self.type.lower() + '-group'.
```

Reply:
I think this makes perfect sense, thanks for pointing out.